code/lib/cli/src/helpers.ts

```diff
@@ -186,6 +186,24 @@ type CopyTemplateFilesOptions = {
   destination?: string;
 };
 
+const frameworkToRenderer: Record<SupportedFrameworks | SupportedRenderers, SupportedRenderers> = {
+  angular: 'angular',
+  ember: 'ember',
+  html: 'html',
+  nextjs: 'react',
+  preact: 'preact',
+  qwik: 'qwik',
+  react: 'react',
+  'react-native': 'react',
+  server: 'react',
+  solid: 'solid',
+  svelte: 'svelte',
+  sveltekit: 'svelte',
+  vue: 'vue',
+  vue3: 'vue',
+  'web-components': 'html',
```

Comment:
We have web-components-specific docs, just saying: https://storybook.js.org/docs/web-components/get-started/install/